specs/xaml-backdrop-api.md

```diff
@@ -0,0 +1,316 @@
+allowing you to simply set a backdrop as a property on a `Window`.
+
+
+# API Pages
+
+## Window.SystemBackdrop property
+
+_This description also applies to `Popup.SystemBackdrop` and `FlyoutBase.SystemBackdrop` properties_
+
+Sets a `SystemBackdrop` to apply that backdrop to this `Window`.
+
+This backdrop is what will render behind the content specified in `Window.Content`.
+If all of the content is fully opaque, this backdrop will have no visible effect.
+
+The following example creates a Window with a Mica backdrop.
+
+```xml
+<Window
+    x:Class="App3.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" >
+
+    <Window.SystemBackdrop>
+        <MicaBackdrop/>
+    </Window.SystemBackdrop>
+
+    <Grid ColumnDefinitions="Auto,*">
+
+        // This area has a transparent background, so the Mica backdrop will be mostly visible
+        // For example if there are buttons here, the backdrop will show up in
+        // the margins and gaps between the button text
+        <local:NavigationControls/>
+
+        // This area has an opaque background, so the Mica backdrop won't be visible
+        <local:ContentArea Background="White"/>
+
+    </Grid>
+</Window>
+```
+
+
+## MicaBackdrop class
+
+_Same description applies to `DesktopAcrylicBackdrop` class_
+
+_See also the base class: `SystemBackdrop`_
+
+Use this class to set a backdrop on a
+[Window](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.Window),
+[Popup](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.Controls.Primitives.Popup),
+or [FlyoutBase](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.Controls.Primitives.FlyoutBase)
+(such as a `Flyout` or `MenuFlyout`).
+
+For example:
+
+```xml
+<Window x:Class="MyApp.MainWindow">
+
+    <Window.SystemBackdrop>
+        <MicaBackdrop MicaKind="BaseAlt"/>
+    </Window.SystemBackdrop>
+
+</Window>
+```
+
+Note that `MicaBackdrop` isn't supported on all systems.
+In such cases a solid color will be used instead of the Mica effect.
+
+
+## SystemBackdrop class
+
+Use this class to create a custom system backdrop.
+You don't create this class directly, but subclass it to add your custom support.
+
+This class is the base of system backdrop classes: `MicaBackdrop` and `DesktopAcrylicBackdrop`.
+
+The following example shows a custom system backdrop class that's implemented using
+[MicaController](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Composition.SystemBackdrops.MicaController).
+
+
+```cs
+public class MicaSystemBackdrop : SystemBackdrop
+{
+    MicaController _micaController;
+
+    protected override void OnTargetConnected(ICompositionSupportsSystemBackdrop connectedTarget, XamlRoot xamlRoot)
+    {
+        base.OnTargetConnected(connectedTarget, xamlRoot);
```

Comment:
This is automatically called internally by Xaml before making the call out to derived class virtual. Explicit call is ok (will no-op) ok but not needed.

Reply:
Recommend: OK to leave here as a practice, but add a comment here and to the doc page that it's not necessary and will be a no op

Reply:
I think the implementation is being updated (post-1.3.0) to require calling base.

Reply:
Addressed in code comment